api/data/scholarly-articles/article1/1.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
all the files in `api/` seem to be for a testing/development configuration of the `dummy-api` container, hence any way to make that more clear? `.api` hidden folder or `.docker` or move in `tests/` or some other subfolder that is clearly not in the application?

Reply:
`.docker` sounds like the best place (used in `dev` only).